### DISCUSSION TITLE
Using SofaContextAPI on android for Quest2

### DISCUSSION THREAD

**Discussion by merandk** [InfinyTech3D/SofaUnity#74] [Help - Setting up SofaUnity]
Hi, I need help with something..
While transferring the scene to the virtual reality glasses (Build and Run), I encounter the following problem: "Assets\SofaUnity\Plugins\SofaUnityAPI\ContextAPI\SofaContextAPI.cs(46,17): error CS0103: The name 'File' does not exist in the current context"

The "SofaContextAPI.cs" file appears to be installed. What path should I follow?
Thanks.

**Comment by epernod** [comment]
Can you precise which version of SofaUnity you are using and for what type of VR headset?
Right now the asset support only windows build, you confirm you are not trying to build for android?

Best regards,

**Comment by merandk** [comment]
Hi,
The trial license defined is SofaAPAPI-Unity3D_v23.12.01. I'm using the VR
headset Oculus Quest 2. I confirm build for Android.

I normally use my Oculus Quest 2 headset on another research project
without any problems. I have no problem transferring scenes.
But I experienced this problem after installing the
SofaAPAPI-Unity3D_v23.12.0.

Best regards,

erik pernod ***@***.***>, 11 Nis 2024 Per, 01:38 tarihinde
şunu yazdı:

**Comment by merandk** [comment]
[Problem is solved]

Best regards,

Merve Karagöz ***@***.***>, 11 Nis 2024 Per, 10:39 tarihinde
şunu yazdı:

**Reply by epernod** [reply]
Hi,

good news, could you explain how you solve the problem to keep a track for the next users?
I'm not sure to understand, you are building and deploying on android? or using the headset connected to your computer?
Did you rebuild sofa for arm64 ?

Vest regards
